index.js

```diff
@@ -86,12 +86,14 @@ function AutoIndex (db, idb, reduce) {
   }
 
   secondary.createValueStream = function (opts) {
-    (opts && opts || (opts = {})).keys = false
+    if (!opts) opts = {}
+    opts.keys = false
     return secondary.createReadStream(opts)
```

Comment:
you want to ensure that (1) opts is an object, and (2) opts.keys is always set to false, right?